pkg/sdkserver/workspaces.go

```diff
@@ -49,8 +52,8 @@ func (s *server) createWorkspace(w http.ResponseWriter, r *http.Request) {
 		prg,
 		s.gptscriptOpts.Env,
 		fmt.Sprintf(
-			`{"provider": "%s"}`,
-			reqObject.ProviderType,
+			`{"provider": "%s", "data_home": "%s", "workspace_ids": "%s"}`,
```

Comment:
Is it actually `data_home`?
I only see it referenced in [Go code](https://github.com/gptscript-ai/workspace-provider/blob/5da7e98d36eabefa2a228ba05ed37a096da071b9/pkg/cli/workspace.go#L14) (not in the tool.gpt), where the env var is [`XDG_DATA_HOME` ](https://github.com/gptscript-ai/workspace-provider/blob/5da7e98d36eabefa2a228ba05ed37a096da071b9/tool.gpt) :thinking:

Reply:
Yes. I just hadn't pushed the changes to the `workspace-provider`. You can look again.